src/udisksdaemonutil.c

```diff
@@ -716,8 +784,9 @@ check_authorization_no_polkit (UDisksDaemon            *daemon,
  * can be used in @message but note that not all variables can be used
  * in all checks. For example, any check involving a #UDisksDrive or a
  * #UDisksBlock object can safely include the fragment
- * <quote>$(drive)</quote> since it will always expand to the name of
- * the drive, e.g. <quote>INTEL SSDSA2MH080G1GC (/dev/sda1)</quote> or
+ * <quote>$(device.name)</quote> since it will always expand to the name of
```

Comment:
Please also update https://github.com/storaged-project/udisks/blob/874b4f0019f08e983c8abbe8b2d05d57a0c55b0d/doc/udisks2-docs.xml.in.in#L221 per the comment above.